src/Microsoft.ML.Data/DataLoadSave/TransformerChain.cs

```diff
@@ -239,7 +239,7 @@ public static class TransformerChain
     {
         public const string LoaderSignature = "TransformerChain";
 
```

Comment:
I think this can be made internal. We usually do that with the `LoaderSignature` elsewhere.

Reply:
No doubt. However I wasn't trying to do that.

---
In reply to: [258701066](https://github.com/dotnet/machinelearning/pull/2649#discussion_r258701066) [](ancestors = 258701066)